src/dex/curve-v1/curve-v1.ts

```diff
@@ -1104,10 +1109,11 @@ export class CurveV1 extends SimpleExchange implements IDex<CurveV1Data> {
             exchange: this.dexKey,
             address: pool.address,
             liquidityUSD: pool.liquidityUSD!,
-            connectorTokens: _.uniq(connectorTokens)
-              .filter(
-                (_token: string) =>
-                  _token.toLowerCase() !== _tokenAddress.toLowerCase(),
```

Comment:
Logic was inversed here

Reply:
It's supposed to be inversed, you are filtering for tokens that DON'T match the input